pyproject.toml

```diff
@@ -66,6 +66,7 @@ markers = [
     # Specify -m weekly_regression to run weekly_regression tests.
     "deepspeed",
     # Whether the test needs to initialize the DeepSpeed engine.
+    "notebooks",
```

Comment:
I'm curious why we need a seperate marker for notebooks? Would it work to mark them as `pytest.mark.timeout(15)` and/or `pytest.mark.gpu`?

Reply:
One notebook still takes ~30-60 seconds, so making them a separate mark for now. It does add length runtime. Some of it is downloding the CIFAR10 dataset..